### PR TITLE
Update readme and documentation

### PR DESCRIPTION
**Pull request by Marinnia**

### CODE REVIEW

**Comment by p5k369** [comment]
Hey @Marinnia, thank you for your pull request.  Would you please run `pre-commit run -a` prior to your next commit, that should solve all minor issues.

File: README.md [hunk 1]
```diff
@@ -33,11 +33,14 @@ It is mainly suited for applications that rely on a center widget for user inter
 - Build-in release history
 - Used and developed in a productive environment
 
-## Themes
+## Themes and Styled Widgets
 
-### Darcula Theme
+QuteStyle provides five themes, defining the color composition of the app.
+Additionally, the user can define new themes ([check this out](./docs/style.md)). We provide five themes, for example a dark and light mode ```Darcula``` and ```Highbridge Grey```.
+We defined [custom widgets](./docs/widgets.md#widgets), such that they fit to the overall style and implemented new behaviour. A selection can be found in the Test-App:
```
Comment (p5k369) [inline README.md:40]:
```suggestion
We defined [custom widgets](./docs/widgets.md), such that they fit to the overall style and implemented new behaviour. A selection can be found in the Test-App:
```

File: docs/qs_main_window.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Main Window
```
Comment (p5k369) [inline docs/qs_main_window.md:1]: Is this a placeholder for information about how to use the main window?

File: docs/widgets.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+```plaintext
+    icon = Icon(18, "background")
+    icon.set_icon(":/path_to_icon.svg")
+````
+
+Attention: The Icon class inherits from QWidget, not QIcon.
+
+### DropLabel
+
+The DropLabel offers a convenient way to display the could icon with an associated text and for example
```
Comment (p5k369) [inline docs/widgets.md:174]:
```suggestion
The DropLabel offers a convenient way to display the cloud icon with an associated text and for example
```

**Review by p5k369 — CHANGES_REQUESTED** [review]

**Review by p5k369 — APPROVED** [review]
Thank you for documenting.

**Review by Daigon2 — APPROVED** [review]

**Pull request merged**